AWSAuthSDK/AWSAuthSDKTestApp/HostedUIUserPoolViewController.swift

```diff
@@ -47,12 +47,18 @@ class HostedUIUserPoolViewController: UIViewController {
     }
 
     private func invalidateRefreshToken() {
```

Comment:
can we add comment to briefly explain this? can link to the SDK code that actually did the check against the expiration of the access token before using the refresh token

Reply:
Added comments